static/js/components/dashboard/CourseAction.js

```diff
@@ -81,63 +85,83 @@ export default class CourseAction extends React.Component {
     }
 
     return (
-      <Button className="dashboard-button" {...buttonProps}>
+      <Button className="dashboard-button" key="1" {...buttonProps}>
         {text}
       </Button>
     );
   }
 
-  renderPayLaterLink(): React$Element<*> {
-    return (
-      <a href="#">Enroll and pay later</a>
-    );
-  }
+  renderDescription = R.curry((className: string, text: string): React$Element<*> => (
+    text.length > 0 ? <div className={className} key="2">{text}</div> : null
```

Comment:
I think the return type on the function inside needs to be changed to `React$Element<*>|null`

alternatively, you can probably leave the annotation off of the function inside `R.curry`.